diff --git a/eix-0.23.3/src/eix-diff.cc b/eix-0.23.3/src/eix-diff.cc
index a6ac580..ee4eb0a 100644
--- a/eix-0.23.3/src/eix-diff.cc
+++ b/eix-0.23.3/src/eix-diff.cc
@@ -40,6 +40,7 @@
 #include <cstddef>
 #include <cstdio>
 #include <cstdlib>
+#include <unistd.h>
 
 #define VAR_DB_PKG "/var/db/pkg/"
 
diff --git a/eix-0.23.3/src/eix.cc b/eix-0.23.3/src/eix.cc
index 9a6f586..462fedf 100644
--- a/eix-0.23.3/src/eix.cc
+++ b/eix-0.23.3/src/eix.cc
@@ -47,6 +47,8 @@
 #include <cstdlib>
 #include <cstring>
 
+#include <unistd.h>
+
 #define VAR_DB_PKG "/var/db/pkg/"
 
 using namespace std;
diff --git a/eix-0.23.3/src/eixTk/md5.cc b/eix-0.23.3/src/eixTk/md5.cc
index dd6bc77..1816741 100644
--- a/eix-0.23.3/src/eixTk/md5.cc
+++ b/eix-0.23.3/src/eixTk/md5.cc
@@ -14,6 +14,7 @@
 #include <sys/mman.h>
 #include <sys/stat.h>
 #include <sys/types.h>
+#include <unistd.h>
 
 using namespace std;
 
