diff -r e5c9898d9ffa backend/dwarf.c
--- a/backend/dwarf.c	Sat Mar 07 13:43:26 2009 +0000
+++ b/backend/dwarf.c	Sat Mar 07 13:49:15 2009 +0000
@@ -51,7 +51,7 @@
 #if ELFOBJ || MACHOBJ
 
 #if MARS
-#include	"../mars/mars.h"
+#include	"../mars.h"
 #endif
 
 #include	"dwarf.h"
diff -r e5c9898d9ffa backend/elfobj.c
--- a/backend/elfobj.c	Sat Mar 07 13:43:26 2009 +0000
+++ b/backend/elfobj.c	Sat Mar 07 13:49:15 2009 +0000
@@ -43,7 +43,7 @@
 #if ELFOBJ
 
 #if MARS
-#include	"../mars/mars.h"
+#include	"../mars.h"
 #endif
 
 #include	"dwarf.h"
diff -r e5c9898d9ffa ph.c
--- a/ph.c	Sat Mar 07 13:43:26 2009 +0000
+++ b/ph.c	Sat Mar 07 13:49:15 2009 +0000
@@ -13,7 +13,7 @@
 #include	<time.h>
 #include	<string.h>
 #include	<stdlib.h>
-#include	<new.h>
+#include	<new>
 
 #include	"cc.h"
 #include	"global.h"
diff -r e5c9898d9ffa tk/mem.c
--- a/tk/mem.c	Sat Mar 07 13:43:26 2009 +0000
+++ b/tk/mem.c	Sat Mar 07 13:49:15 2009 +0000
@@ -15,7 +15,7 @@
 #include	<stddef.h>
 
 #if __cplusplus
-#include	<new.h>
+#include	<new>
 #endif
 
 #ifndef malloc
